src/lib/block/seed/seed.cpp

```diff
@@ -173,9 +169,7 @@ void SEED::key_schedule(std::span<const uint8_t> key) {
 
    secure_vector<uint32_t> WK(4);
 
-   for(size_t i = 0; i != 4; ++i) {
-      WK[i] = load_be<uint32_t>(key.data(), i);
-   }
+   load_be(key.first<BLOCK_SIZE>(), WK[0], WK[1], WK[2], WK[3]);
```

Comment:
With https://github.com/randombit/botan/pull/3869 this could become:

```C++
auto WK = load_be<secure_vector<uint32_t>>(key.first<BLOCK_SIZE>());
```